go.mod

```diff
@@ -1,20 +1,23 @@
 module tinygo.org/x/drivers
 
-go 1.18
+go 1.22.1
```

Comment:
It would be better to require the minimum version of Go supported by TinyGo here instead. Which would be current version minus 2 e.g. Go 1.21